src/deroff.rs

```diff
@@ -530,6 +533,27 @@ impl Deroffer {
     }
 }
 
+fn deroff_files(files: &[String]) -> std::io::Result<()> {
+    for arg in files {
+        eprintln!("{}", arg);
+
+        let mut file = File::open(arg)?;
+        let mut string = String::new();
+        if arg.ends_with(".gz") {
+            let mut decoder = Decoder::new(file).unwrap();
+            decoder.read_to_string(&mut string);
+        } else {
+            file.read_to_string(&mut string)?;
+        }
+        let d = Deroffer::new();
+        println!("string: {}", string);
+
+        // d.deroff(string);
```

Comment:
Perhaps a `// TODO Enable when implemented` comment?

Reply:
For type-checking could maybe at least add in the function type signatures but leave the body's `unimplemented!()`?

Reply:
added!